db/migrate/20230116201724_refactor_group_membership.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class RefactorGroupMembership < ActiveRecord::Migration[6.1]
+  def up
+    rename_table :group_memberships, :group_memberships_old
+
+    create_table :group_memberships, id: :uuid, force: :cascade do |t|
+      t.belongs_to :user, foreign_key: true, null: false, index: true
+      t.belongs_to :group, foreign_key: true, null: false, index: true
+      t.integer :role, limit: 1, null: false, default: 0, comment: 'Used as enum in Rails'
+      t.timestamps
+    end
+
+    GroupMembershipOld.where(member_type: 'User').and(GroupMembershipOld.where.not(membership_type: nil)).each do |gmo|
```

Comment:
This is equivalent to 
```suggestion
    GroupMembershipOld.where(member_type: 'User').where.not(membership_type: nil).each do |gmo|
```

I changed it accordingly in 514f2c27.